Error: Cannot find module './Router'

Hi Alex, 

I'm trying to run your code but I'm failing to start the service. It fails with the following error. 
Have you got an idea of what could be the reason ? 

Thanks a lot !

-------

pi@raspberrypi:~/ln-service $ npm start

\> ln-service@4.0.0 start /home/pi/ln-service
\> node server.js

internal/modules/cjs/loader.js:573
    throw err;
    ^

Error: Cannot find module './Router'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:571:15)
    at Function.Module._load (internal/modules/cjs/loader.js:497:25)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/pi/ln-service/routers/payments.js:3:16)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ln-service@4.0.0 start: `node server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ln-service@4.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2018-05-23T08_32_38_844Z-debug.log
pi@raspberrypi:\~/ln-service $
pi@raspberrypi:\~/ln-service $ npm -v
6.0.1
pi@raspberrypi:\~/ln-service $ node -v
v10.1.0

--- a/routers/payments.js
+++ b/routers/payments.js
@@ -1,6 +1,6 @@
 const {payInvoice} = require('./../lightning');
 const {returnJson} = require('./../async-util');
-const Router = require('./Router');
+const Router = require('./router');
 
 /** Get a payments router.
 

